md5.js

```diff
@@ -187,20 +185,19 @@ need the idiotic second function,
 generated by an if clause.  */
 
 function add32(a, b) {
-return (a + b) & 0xFFFFFFFF;
+  return (a + b) & 0xFFFFFFFF;
 }
 
 if (md5('hello') != '5d41402abc4b2a76b9719d911017c592') {
-function add32(x, y) {
-var lsw = (x & 0xFFFF) + (y & 0xFFFF),
-msw = (x >> 16) + (y >> 16) + (lsw >> 16);
-return (msw << 16) | (lsw & 0xFFFF);
-}
+  function add32(x, y) {
```

Comment:
Function declarations should not be placed in blocks. Use a function expression or move the statement to the top of the outer function.<br>'add32' is already defined.